README.md

```diff
@@ -1,3 +1,8 @@
+## Usage at Snapdocs
+This fork version is in use at [monolith scheduling tasks](https://github.com/snapdocs/snapdocs/blob/ec8663419cf94df30b89ea81f63dede1fdd2174f/lib/tasks/one_time_tasks.rake#L226)
```

Comment:
Might also be worth linking to the Gemfile that shows this fork is used instead of the original? https://github.com/snapdocs/snapdocs/blob/ec8663419cf94df30b89ea81f63dede1fdd2174f/Gemfile#L435

Reply:
I tried to update this comment, but I apparently don't have push access to this repo. So here's a suggestion for the change instead:

```suggestion
This fork version is [used by the monolith](https://github.com/snapdocs/snapdocs/blob/ec8663419cf94df30b89ea81f63dede1fdd2174f/Gemfile#L435) to [hide obscene comments](https://github.com/snapdocs/snapdocs/blob/869e3997a6e95b820da3b04505d52f9347a07d6c/app/models/comment.rb#L200)
```